plugins/search/src/apis.ts

```diff
@@ -39,8 +39,8 @@ class SearchApi {
       name: result.metadata.name,
       description: result.metadata.description,
       owner: result.spec.owner,
-      kind: result.kind,
-      lifecycle: result.spec.lifecycle,
+      kind: '',
```

Comment:
We do want to set kind and lifecycle here to what is returned from the catalog api request, so back to how it was before I guess. Any reason to why it was replaced by empty strings?

Reply:
Ops, that's my bad, I replaced it to empty strings to check, when there are no values and forgot to bring it back. Will update it